bitcoin/src/network/message.rs

```diff
@@ -672,7 +682,8 @@ mod test {
             0xfa, 0xa9, 0x95, 0x59, 0xcc, 0x68, 0xa1, 0xc1,
             0x10, 0x2f, 0x53, 0x61, 0x74, 0x6f, 0x73, 0x68,
             0x69, 0x3a, 0x30, 0x2e, 0x31, 0x37, 0x2e, 0x31,
-            0x2f, 0x93, 0x8c, 0x08, 0x00, 0x01, 0, 0 ];
+            0x2f, 0x93, 0x8c, 0x08, 0x00, 0x01, 0,    0
```

Comment:
I think the trailing `0`s should be `0x00`s for readability.

Reply:
Certainly - done, thanks man!